apps/dashboard/app/apps/manifest.rb

```diff
@@ -65,6 +65,10 @@ def initialize(opts)
     raise InvalidContentError.new unless(opts && opts.respond_to?(:to_h))
 
     @manifest_options = opts.to_h.with_indifferent_access
+
+    @manifest_options.select! do |method| 
```

Comment:
Why not continue to chain? mutating a member variable a second time seems like a code smell for Ruby.

```suggestion
    @manifest_options = opts.to_h.with_indifferent_access.select! do |method| 
```

Reply:
Also I think you want the second parameter there, `_val`. Looping through a single variable means you're looping through keys and values pairs. 

```irb
irb(main):004:0> {a: 'bar'}.each { |p| puts p.inspect }
[:a, "bar"]
```

I'm actually not sure how this works with only 1 parameter to the block.
```irb
irb(main):003:0> {a: 'bar', to_s: 'foo' }.each { |m, val| puts self.respond_to?(m) }
false
true
=> {:a=>"bar", :to_s=>"foo"}
irb(main):004:0> {a: 'bar', to_s: 'foo' }.each { |m| puts self.respond_to?(m) }
Traceback (most recent call last):
        7: from /home/jeff/apps/ruby/2.7.6/bin/irb:23:in `<main>'
        6: from /home/jeff/apps/ruby/2.7.6/bin/irb:23:in `load'
        5: from /home/jeff/apps/ruby/2.7.6/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        4: from (irb):4
        3: from (irb):4:in `each'
        2: from (irb):4:in `block in irb_binding'
        1: from (irb):4:in `respond_to?'
TypeError ([:a, "bar"] is not a symbol nor a string)


irb(main):003:0> {a: 'bar', to_s: 'foo' }.each { |m, val| puts self.respond_to?(m) }
false
true
```

Reply:
Also I think you want the second parameter there, `_val`. Looping through a single variable means you're looping through keys and values pairs. 

```irb
irb(main):004:0> {a: 'bar'}.each { |p| puts p.inspect }
[:a, "bar"]
```

I'm actually not sure how this works with only 1 parameter to the block.
```irb
irb(main):003:0> {a: 'bar', to_s: 'foo' }.each { |m, val| puts self.respond_to?(m) }
false
true
=> {:a=>"bar", :to_s=>"foo"}
irb(main):004:0> {a: 'bar', to_s: 'foo' }.each { |m| puts self.respond_to?(m) }
Traceback (most recent call last):
        7: from /home/jeff/apps/ruby/2.7.6/bin/irb:23:in `<main>'
        6: from /home/jeff/apps/ruby/2.7.6/bin/irb:23:in `load'
        5: from /home/jeff/apps/ruby/2.7.6/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        4: from (irb):4
        3: from (irb):4:in `each'
        2: from (irb):4:in `block in irb_binding'
        1: from (irb):4:in `respond_to?'
TypeError ([:a, "bar"] is not a symbol nor a string)


irb(main):003:0> {a: 'bar', to_s: 'foo' }.each { |m, val| puts self.respond_to?(m) }
false
true
```